features/FxA-57-verify-reminder/README.md

```diff
@@ -12,7 +12,17 @@ As a new Firefox Accounts user,
  The first email arrives after 48 hours since sign up.
  
  The second email arrives after 168 hours since sign up.
- 
+
+## Success criteria
+
+We are planning to get the results from the following metrics:
+
+- Measure number of people that verify using the first email and second email.
+We would use the total number of emails sent vs verified.
```

Comment:
Channeling @ckarlof, let's be very explicit that we'll measure it "as a percentage of number of emails sent".